test/interface.jl

```diff
@@ -0,0 +1,242 @@
+using Test
+using Bijectors
+using Random
+using LinearAlgebra
+
+Random.seed!(123)
+
+# Scalar tests
+@testset "Interface" begin
+    @testset "Univariate" begin
+        # Tests with scalar-valued distributions.
+        uni_dists = [
+            Arcsine(2, 4),
+            Beta(2,2),
+            BetaPrime(),
+            Biweight(),
+            Cauchy(),
+            Chi(3),
+            Chisq(2),
+            Cosine(),
+            Epanechnikov(),
+            Erlang(),
+            Exponential(),
+            FDist(1, 1),
+            Frechet(),
+            Gamma(),
+            InverseGamma(),
+            InverseGaussian(),
+            # Kolmogorov(),
+            Laplace(),
+            Levy(),
+            Logistic(),
+            LogNormal(1.0, 2.5),
+            Normal(0.1, 2.5),
+            Pareto(),
+            Rayleigh(1.0),
+            TDist(2),
+            TruncatedNormal(0, 1, -Inf, 2),
+        ]
+        
+        for dist in uni_dists
```

Comment:
Have we tested if the inverse function can map things back. Something like
```julia
y = rand(td)
x = inv(td.transform)(y)
@test y == td.transform(x)
```

Reply:
Good catch; we should have such a test